### ISSUE TITLE
Could not find enough space to create new chunk. Try clearing a few gigabytes of space, or defragging.

### ISSUE THREAD

**Issue by solarisfire** [maharmstone/ntfs2btrfs#70]
Yes it's a 2TB drive that's 91% full, but is 174G not enough space to do an "in place" conversion in??

/dev/sdc2       1.9T  1.7T  174G  91% /media/ssd1

And could it not throw this error before it sits there for almost an hour calculating checksums?

**Comment by maharmstone** [comment]
It needs contiguous space - I'm guessing you've got lots of part-allocated chunks. What does `btrfs fi usage` say?

**Comment by solarisfire** [comment]
No, the drive isn't compressed at all... It contains a lot of Steam Games, and some Windows Apps that I really don't care about...

**Comment by RayJW** [comment]
Were you able to resolve this? I ran into the same issue on a 4 TB drive with over 500 GB of free space, I think that should be enough to convert the drive, no?

**Comment by cpanceac** [comment]
See also: https://bugzilla.redhat.com/show_bug.cgi?id=2238054.
Not sure what is the right answer but for a EXT4 to BTRFS conversion 4% of free space was enough.
So, for my 500 GB NTFS partition i'll try 20 GB.
Once the next test is completed i'll report back.
ETA tomorrow evening :)

**Comment by maharmstone** [comment]
There's really no point in logging it with Red Hat, all they're going to do is send it back to me...

**Comment by cpanceac** [comment]
Thank you, new issue created: https://github.com/maharmstone/ntfs2btrfs/issues/72

**Comment by cpanceac** [comment]
To all interested parties: to me 4% free space was enough to complete the conversion.
But maybe it depends on the type of files stored or other things.

**Comment by antiman0** [comment]
Having this issue on Windows, converting external drive for SteamOS to BTRFS from NTFS. It's not even 75% used and it still shows this error.
